### ISSUE TITLE
Unreadable text colors on MDN Plus pages with dark theme

### ISSUE THREAD

**Issue by caugner** [mdn/fred#545]
### What information was incorrect, unhelpful, or incomplete?

On https://fred.review.mdn.allizom.net/en-US/plus/settings, the following texts are not readable:

- "Don't want to see ads?" (in the top banner)
- Theme / Language switcher (in the breadcrumbs bar)
<img width="898" height="146" alt="Image" src="https://github.com/user-attachments/assets/9e98371f-1bd3-486c-9c78-6184269f50c1" />

### What did you expect to see?

On https://developer.allizom.org/en-US/plus/settings, "Don't want to see ads?" is readable (and there is no breadcrumbs bar).

<img width="898" height="146" alt="Image" src="https://github.com/user-attachments/assets/1570aa18-40be-4ed2-97b9-23085d20f4b0" />

**Comment by LeoMcA** [comment]
> "Don't want to see ads?" (in the top banner)

Was fixed by https://github.com/mdn/fred/pull/566

> Theme / Language switcher (in the breadcrumbs bar)

Not sure what this was fixed by, but works for me now.

> (and there is no breadcrumbs bar).

Showing the breadcrumbs bar everywhere was an explicit change we made in fred: why would we e.g. hide the theme switcher on the settings?

I think this is fixed, please reopen if I've missed something.
